llvm/lib/RemoteCachingService/Client/Client.cpp

```diff
@@ -38,6 +38,119 @@ static Error errorFromGRPCStatus(const grpc::Status &Status) {
   return createStringError(inconvertibleErrorCode(), Status.error_message());
 }
 
+static Expected<Optional<KeyValueDBClient::ValueTy>>
+createGetValueRespond(const GetValueResponse &Resp) {
```

Comment:
`createGetValueRespond` -> `createGetValueResponse`? Same for the other functions below.